src/parser/ParsedQuery.cpp

```diff
@@ -609,6 +615,9 @@ void GraphPatternOperation::toString(std::ostringstream& os,
         os << arg._whereClauseTriples.back().asString();
       }
 
+    } else if constexpr (std::is_same_v<T, Bind>) {
+      os << "Some kind of BIND\n";
+      // TODO<joka921> proper ToString (are they used for something?)
```

Comment:
They were used for the unit tests. There are still some references from there to this `toString` method, but we could probably get rid of it.

Reply:
Yes, they are basically unused. I have made this an issue and will remove it in a future PR.